.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  parser: 'babel-eslint',
+  extends: 'airbnb',
```

Comment:
we should extend https://github.com/cssinjs/eslint-config-jss

Reply:
Okay, but let's update the packages before because it's really outdated

Reply:
sure